### PR TITLE
[JENKINS-68985] Context menu shadow looks terrible on redesigned "About Jenkins" page

### PR DESCRIPTION
**Pull request by frankie139506**
See [JENKINS-68985](https://issues.jenkins.io/browse/JENKINS-68985).

image

### Proposed changelog entries

* Make context menu shadow darker than before.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] (If applicable) Jira issue is well described
- [X] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [X] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")` if applicable.
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/6838"></a>

### CODE REVIEW

**Review by NotMyFault — APPROVED** [review]
Looks good

**Comment by daniel-beck** [comment]
I wonder whether the same "color" could be kept as almost-black + opacity? That way it even works when the background isn't black (while not appearing darker, as this PR).

File: war/src/main/less/abstracts/theme.less
Comment (daniel-beck) [inline war/src/main/less/abstracts/theme.less:283]: While this resolves the reported issue, the shadow is a lot darker than before.

I wonder whether the same effect on About Jenkins can be achieved by having a shadow color that's basically black + opacity. Something like this?

```suggestion
  --menu-box-shadow: 0 3px 10px rgba(0,0,0,0.3);
```

Comment (frankie139506) [reply war/src/main/less/abstracts/theme.less]: Need to add a space after "," or you can't run. I have made some change. please take a look.

Comment (frankie139506) [reply war/src/main/less/abstracts/theme.less]: > Need to add a space after "," or you can't run. I have made some change, please take a look.

**Review by janfaracik — APPROVED** [review]

**Review by daniel-beck — APPROVED** [review]
This is great, thanks!
Screenshot 2022-07-14 at 08 31 41 Screenshot 2022-07-14 at 08 31 26

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Comment by daniel-beck** [comment]
> [[JENKINS-68913] Bottom bar for update center "Updates" view](https://github.com/jenkinsci/jenkins/pull/6838/commits/11345540139711dddd3d1f8140c0bef312392c8b)

Misleading commit message now in the `master` history 😢

**Pull request merged**
